### ISSUE TITLE
Improve suppress author behavior

### ISSUE THREAD

**Issue by bdarcus** [andras-simonyi/citeproc-el#19]
Am I correct the behavior we discussed in this and subsequent two posts is not implemented here?

https://lists.gnu.org/archive/html/emacs-orgmode/2021-04/msg00554.html

Explained with examples here:

https://lists.gnu.org/archive/html/emacs-orgmode/2021-04/msg00572.html

Though updated syntax would be:

``` org
[cite/noauthor:@doe20; @doe21]
[cite/noauthor:@doe20; see also @smith19]
[cite/text:@doe20; see also @smith19]
```

From your explanation in that first post above @andras-simonyi, it sounds like you implement it in citeproc-org, but not here, which seems the more obvious place.

And I guess to do that, you need to add a parameter to the citation, whereas now it's on the individual cites?

Obviously this will also relate to #15.

cc @denismaier

**Comment by andras-simonyi** [comment]
Regarding this: is it intentional that noauthor doesn't appear on the [bibtex-actions style names and mapping page](https://github.com/bdarcus/bibtex-actions/wiki/Org-cite)?

**Comment by bdarcus** [comment]
> Regarding this: is it intentional that `noauthor` doesn't appear on the [bibtex-actions style names and mapping page](https://github.com/bdarcus/bibtex-actions/wiki/Org-cite)?

No; I put that together before that became a style.

I'll try to update it this week sometime.

**Comment by andras-simonyi** [comment]
Closing this as the branch with the suggested improvement has been merged, see #29.
